solr/modules/ltr/src/test/org/apache/solr/ltr/store/rest/TestManagedFeatureStore.java

```diff
@@ -103,23 +103,38 @@ public void testFeatureStoreGet() throws FeatureException {
     }
   }
 
+  @Test
+  public void testWrongFeatureStore() {
```

Comment:
try to follow the same naming convention for tests:
functionalArea_status_expectedOutcome
e.g.
featureExtraction_valueFeatureRequiredInFq_shouldThrowException